### PR TITLE
Feature/user profile route

### PR DESCRIPTION
**Pull request by Willcowen**
Added header component to profile form, corrected dependancy array.

### CODE REVIEW

**Comment by marianphillips** [comment]
I think there may be an issue now (sorry I asked about this earlier) where you have to pass a companyname prop. That was entirely our fault and we should fix that before this PR is submitted, to make the header better. Sorry again because I asked about this earlier and only just realised.

File: src/components/users/userProfile/UserProfile.js [hunk 1]
```diff
@@ -28,6 +29,7 @@ const UserProfile = () => {
 
   return (
     <div>
+      < Header />
```
Comment (mikemherron) [inline src/components/users/userProfile/UserProfile.js:32]: Would remove this leading space.

**Review by mikemherron — DISMISSED** [review]
Tiny formatting suggestion

File: src/components/users/userProfile/UserProfile.js [hunk 2]
```diff
@@ -15,26 +16,24 @@ const UserProfile = () => {
...
-    })
-    .catch(err => console.log(err.response))
-  };
+      client
+        .get(`/user/${id}`)
+        .then((res) => {
+          setProfile(res.data.data.user);
+        })
+        .catch((err) => console.log(err.response));
+  }, []);
```
Comment (mikemherron) [inline src/components/users/userProfile/UserProfile.js:25]: The cypress tests are failing due to this warning:

>  Line 25:6:  React Hook useEffect has a missing dependency: 'id'. Either include it or remove the dependency array  react-hooks/exhaustive-deps

I think you'll need to resolve that warning before the build passes and you can merge.

File: src/components/users/userProfile/UserProfile.js [hunk 3]
```diff
@@ -15,26 +16,24 @@ const UserProfile = () => {
...
 
   return (
     <div>
+      <Header/>
```
Comment (mikemherron) [inline src/components/users/userProfile/UserProfile.js:29]: Not for now, but if every route needs to include this header component - there is probably a better way we could structure the overall app. Something to think about next week for the teams. 

Also - see @marianphillips comment - do you need to pass in a prop here?

**Review by mikemherron — APPROVED** [review]

**Review by phillipdunn — APPROVED** [review]
LGTM

**Comment by Willcowen** [comment]
> LGTM - need to fix cypress

I'm not sure whats causing this, i'll look into it.

**Review by nicopicchio — CHANGES_REQUESTED** [review]
Hey Will, I have noticed you commented out the `initialProfileState` variable. Since it's not being used can we remove it before committing to `main`?

**Comment by Willcowen** [comment]
> Hey Will, I have noticed you commented out the `initialProfileState` variable. Since it's not being used can we remove it before committing to `main`?

Yeah, this has been done, apologies!

**Review by nicopicchio — APPROVED** [review]
LGTM

**Review by mikemherron — APPROVED** [review]

**Pull request merged**